python/ray/data/tests/test_csv.py

```diff
@@ -846,7 +813,7 @@ def test_csv_read_with_column_type_specified(shutdown_only, tmp_path):
     Version(pa.__version__) < Version("7.0.0"),
     reason="invalid_row_handler was added in pyarrow 7.0.0",
 )
-def test_csv_invalid_file_handler(shutdown_only, tmp_path):
+def test_csv_invalid_file_handler(ray_start_regular_shared, tmp_path):
```

Comment:
should we update other tests to use `ray_start_regular_shared` instead of `shutdown_onl`? e.g. `test_csv_read_with_column_type_specified`

Reply:
Yeah, we should. Can do in a follow-up PR if/when I have time

Reply:
Follow-up here: https://github.com/ray-project/ray/pull/48352